tests/jwt_test.py

```diff
@@ -28,13 +28,20 @@ def test_refresh_jwt_issuer(network, args):
     return network
 
 
+def make_pem_cert(pem_encoded):
```

Comment:
Can't we just re-use the existing `infra.crypto.cert_der_to_pem` function?

Reply:
It's not particularly convenient, we have to decode the string first. If anything I was more leaning towards making the endpoint do the wrapping so we return proper PEM certs.

Reply:
Returning PEM directly sounds good!

Reply:
+1 on PEM, as that allows to serve public key PEMs as well (if they should come in the future, see #2671) and keep a single response schema,

Reply:
Yes I've done that now.